src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" data-qa="big">
+      <img
+        src="images/Search.svg"
+        alt="image"
+        class="bigSearch-svg"
+      >
+      <input
+        type="text"
+        name="bigSearch"
+        data-qa="keypress"
+        class="bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    <form action="#" data-qa="small">
```

Comment:
Check it everywhere, please
```suggestion
    </form>
    
    <form action="#" data-qa="small">
```

Reply:
commited suggestion